dune-project

```diff
@@ -22,4 +22,4 @@
   containers
   fmt
   (cmdliner (>= 1.1.0))
-  (ocamlformat (and :with-dev-setup (= 0.26.2)))))
+  (ocamlformat (and :with-dev-setup (= 0.26.2)))))
```

Comment:
You removed the newline at the end of the file. This file shouldn't be touched by this PR at all.

Reply:
Hello sir @NathanReb, if i understood you clearly does it mean i have to add a newly at the end of the file you mentioned above?

Reply:
You shouldn't modify this file as part of this PR. You can revert it to its original state. There are various ways you can do this but for now let's stick to the simplest:
```
git fetch --all
git reset --hard origin/main dune-project
```